app/models/invoice.rb

```diff
@@ -13,7 +13,7 @@ class Invoice < ApplicationRecord
   }
 
   def pretty_created_at
-    created_at.strftime("%A, %B%-d, %Y")
+    created_at.strftime("%A, %B %-d, %Y")
```

Comment:
I didn't catch this on the last pull request review.  I'm sure its something you noticed when you looked at the page, which I didn't actually do!